src/App.css

```diff
@@ -165,6 +165,10 @@ li.panel .button-list {
   .panel {
     padding: 0;
   }
+  .nav__item {
```

Comment:
can we use the template css instead ?

Reply:
The template uses single word name for their `nav` and the background color is equal to the active color.
If we change the template, this is what we get
![image](https://user-images.githubusercontent.com/13916213/45481507-fecbcf00-b74b-11e8-9160-f4397c3e6c56.png)